app/scripts/modules/core/src/utils/debug.ts

```diff
@@ -22,7 +22,7 @@ const getMethodName = (target: any, propertyKey: string) => {
 export const DebugTiming = (label?: string) => (target: any, propertyKey: string, descriptor: PropertyDescriptor) => {
   const fn = descriptor.value;
   label = padStart(label ? label : getMethodName(target, propertyKey), 50);
-  descriptor.value = function DebugTiming() {
+  descriptor.value = function() {
```

Comment:
In general I tend to add named functions when I expect to look for them in a stack trace.  

LGTM though, shouldn't be a problem.